### PR TITLE
zh: add a simpe desc about HA

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

- add a simpe desc about HA

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@lance6716 @ireneontheway PTAL

**Comment by lance6716** [comment]
LGTM

**Review by ireneontheway — COMMENTED** [review]
LGTM @yikeke

File: zh/overview.md [hunk 1]
```diff
@@ -55,6 +55,12 @@ dmctl 是用来控制 DM 集群的命令行工具。
 - 处理数据同步任务错误
 - 校验数据同步任务配置的正确性
 
+### 高可用
+
+当部署多个 DM-master 节点时，所有 DM-master 节点将使用内部嵌入的 etcd 组成集群并用于存储集群节点信息、任务配置等元数据，同时通过 etcd 选举出 leader 节点用于提供集群管理、数据迁移任务管理相关的各类服务。因此，若 DM-master 可用节点数超过部署节点的半数，即可正常提供服务。
```
Comment (yikeke) [inline zh/overview.md:60]:
```suggestion
当部署多个 DM-master 节点时，所有 DM-master 节点将使用内部嵌入的 etcd 组成集群。该 DM-master 集群用于存储集群节点信息、任务配置等元数据，同时通过 etcd 选举出 leader 节点。该 leader 节点用于提供集群管理、数据迁移任务管理相关的各类服务。因此，若可用的 DM-master 节点数超过部署节点的半数，即可正常提供服务。
```
明确下各个句子的主语

File: zh/overview.md [hunk 2]
```diff
@@ -55,6 +55,12 @@ dmctl 是用来控制 DM 集群的命令行工具。
...
+
+当部署的 DM-worker 节点数超过上游 MySQL/MariaDB 节点数时，超出上游节点数的相关 DM-worker 节点默认将处于空闲状态。若某个 DM-worker 节点下线或与 DM-master leader 发生网络隔离，DM-master 能自动将与原 DM-worker 节点相关的数据迁移任务调度到其他空闲的 DM-worker 节点上（若原 DM-worker 节点为网络隔离状态，则其会自动停止相关的数据迁移任务）；若无空闲的 DM-worker 节点可供调度，则原 DM-worker 相关的数据迁移任务将无法进行。
```
Comment (yikeke) [inline zh/overview.md:62]:
```suggestion
当部署的 DM-worker 节点数超过上游 MySQL/MariaDB 节点数时，超出上游节点数的相关 DM-worker 节点默认将处于空闲状态。若某个 DM-worker 节点出现异常，如非正常下线或与 DM-master leader 发生网络隔离，DM-master 能自动将与异常 DM-worker 节点相关的数据迁移任务调度到其他空闲的 DM-worker 节点上（若原 DM-worker 节点为网络隔离状态，则其会自动停止相关的数据迁移任务）；若无空闲的 DM-worker 节点可供调度，则原 DM-worker 相关的数据迁移任务将无法进行。
```
s/原/异常

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**